chesster.js

```diff
@@ -1169,3 +1177,127 @@ controller.hears([
         }
     });
 });
+
+
+/* Scheduling */
+
+// Scheduling reply helpers
+
+// Can't find the pairing
+function scheduling_reply_missing_pairing(bot, message) {
+    var user = "<@"+message.user+">";
+    bot.reply(message, ":x: " + user + " I couldn't find your pairing. Please use a format like: @lakinwecker v @lakinwecker 04/16 @ 16:00 GMT");
+}
+
+// you are very close to the cutoff
+function scheduling_reply_close_to_cutoff(bot, message, scheduling_options, white, black) {
+    bot.reply(message, 
+        ":heavy_exclamation_mark: @" + white.name + " " + "@" + black.name + " " + scheduling_options.warning_message
+    );
+}
+
+// Game has been scheduled.
+function scheduling_reply_scheduled(bot, message, results, white, black) {
+    var whiteDate = results.date.clone().utcOffset(white.tz_offset/60);
+    var blackDate = results.date.clone().utcOffset(black.tz_offset/60);
+    var format = "YYYY-MM-DD @ HH:mm";
+    var dates = [
+        results.date.format(format) + " in UTC",
+        whiteDate.format(format + " ZZ") + " for " + white.name,
+        blackDate.format(format + " ZZ") + " for " + black.name,
+    ];
+    date_formats  = dates.join("\n\t");
+
+    bot.reply(message, 
+        ":heavy_check_mark: @" + white.name + " (white pieces) vs " + "@" + black.name + " (black pieces) scheduled for: \n\t" + date_formats
+    );
+}
+
+
+// Can't parse the date
+function scheduling_reply_cant_parse(bot, message) {
+    var user = "<@"+message.user+">";
+    bot.reply(message, ":x: " + user + " I don't understand. Please use a format like: @lakinwecker v @lakinwecker 04/16 @ 16:00 GMT");
```

Comment:
I'm probably being silly, but this may be misleading.

I'll probably change this to say: @white v @black 04/16 @ 16:00